app/helpers/comments_helper.rb

```diff
@@ -1,13 +1,14 @@
 module CommentsHelper
-	def comments_json(commentable_object)
-		if commentable_object
-			eval(ActiveModel::ArraySerializer.new(commentable_object.comments.recent_last, each_serializer: CommentSerializer).to_json)
-		end
-	end
+  def comments_json(commentable_object)
+    if commentable_object
+      eval(ActiveModel::ArraySerializer.new(commentable_object.comments.recent_last,
+                                            each_serializer: CommentSerializer).to_json)
+    end
+  end
 
-	def current_user_json
-		if current_user
-			eval(UserSerializer.new(current_user).to_json)
-		end
-	end
+  def current_user_json(user = current_user)
```

Comment:
by "external infrastructure" I mean something like a global current_user variable.  If your helper requires that, it makes them hard to test.